gpustat/core.py

```diff
@@ -487,11 +493,12 @@ def print_formatted(self, fp=sys.stdout, force_color=False, no_color=False,
             raise ValueError("--color and --no_color can't"
                              " be used at the same time")
 
-        if force_color:
+        if force_color and not self.is_windows():
```

Comment:
Why checking windows here? On windows, the `--color` flag should also work.

Reply:
you're right! my bad